test/__snapshots__/TextForEvent-test.ts.snap

```diff
@@ -0,0 +1,113 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
this file doesn't have an immediate purpose in this project.

Reply:
IMHO we should definitely add snapshot tests when adding new stuff, if we won't start doing it at some point we'll never have snapshots coverage.

Reply:
AFAICT, this file is used by `jest` to make sure the produced `JSX` is as expected:

```js
expect(component.toJSON()).toMatchSnapshot();
```

Reply:
> AFAICT, this file is used by `jest` to make sure the produced `JSX` is as expected:
> 
> ```js
> expect(component.toJSON()).toMatchSnapshot();
> ```
> 
> I'm not too familiar with snapshots though, so I could be wrong.

You're right, it's a thing that jest does by itself when you're using snapshots so that you can compare those in CI. A very simple regression test basically.

Reply:
if we want to add snapshots then we should do that in its own PR.

Reply:
or, hmm, I guess it's a thing based on the match conditions in this PR. Typically our tests don't do snapshots, so I'd be hesitant to include it here. Can we test the component in some other way for now?

Reply:
I refactored the tests so that snapshots aren't used, by using a custom function that renders the component to a plain text string.